### ISSUE TITLE
Specify the location of the Web Application Manifest

### ISSUE THREAD

**Issue by reillyeon** [WICG/isolated-web-apps#24]

**Comment by mgiuca** [comment]
It seems that it's hard-coded to be at `/manifest.webmanifest` (which is the example name given in the [Manifest spec](https://www.w3.org/TR/appmanifest/#example-linking-to-a-manifest)).

Consider moving it to `/.well-known/manifest.webmanifest`. Per [RFC 8615](https://datatracker.ietf.org/doc/html/rfc8615), to avoid "usurp[ing] the origin's control over its own URI space", resources that are mandated by a spec to live in a precise URL should be within the `/.well-known/` directory.

In a private discussion with someone internally (anonymous since it was a private discussion), they said: "[Changing it] would make things more difficult for developers, especially during the transition period where Chrome and the packaging tools we made disagree about where it should go." I would suggest that while standards are still being prototyped and have not been released on any browser, we should be amenable to making design changes even if they disrupt developers. If necessary, Chrome could have a grace period where it looks at the new location and falls back to looking at the old location with a warning.

**Comment by scheib** [comment]
Seams reasonaable. .well-known, though proposed in 2019, has not progressed to best practice or standard yet. (Note, I'm not familiar with IETF context to understand why). [MDN doesn't appear to have use of well-known](https://developer.mozilla.org/en-US/search?q=%22.well-known%22), I didn't find W3C standards yet using it (difficult to search), though I did find a list at https://en.wikipedia.org/wiki/Well-known_URI

**Comment by reillyeon** [comment]
I think the only example I've seen of it in a proposed web standard is in First Party Sets. I'm not opposed to migrating.

**Comment by mgiuca** [comment]
Good point that `.well-known` may not be very prominent in standards yet.

The Wikipedia page suggests quite a few official IETF uses including http-opportunistic in HTTP/2. FWIW we have proposed using it for [Scope Extensions](https://github.com/WICG/manifest-incubations/blob/gh-pages/scope_extensions-explainer.md) in Manifest itself which is quite close to home.

**Comment by HughIsaacs2** [comment]
> Seems reasonable. .well-known, though proposed in 2019, has not progressed to best practice or standard yet. (Note, I'm not familiar with IETF context to understand why). [MDN doesn't appear to have use of well-known](https://developer.mozilla.org/en-US/search?q=%22.well-known%22), I didn't find W3C standards yet using it (difficult to search), though I did find a list at https://en.wikipedia.org/wiki/Well-known_URI

Might be an issue with MDN search because it's definitely in this FedCM article.
https://developer.mozilla.org/en-US/docs/Web/API/FedCM_API#provide_a_well-known_file

**Comment by robbiemc** [comment]
Oops, #34 does not fix this :)
